tests/test_dragon_launcher.py

```diff
@@ -0,0 +1,75 @@
+import typing as t
+
+import pytest
+
+from smartsim._core.launcher.dragon.dragonLauncher import DragonLauncher
+from smartsim._core.schemas.dragonRequests import DragonBootstrapRequest
+from smartsim.error.errors import LauncherError
+
```

Comment:
Similar request here: can we add a:
```py
# The tests in this file belong to the group_a group
pytestmark = pytest.mark.group_a
```
or similar?

Reply:
(Fun side conversation for those interested) @MattToast @ankona @al-rigazzi 

Should we have a CI check make sure that pytest collects the same number of tests under "normal" and "grouped" execution? At this point I'm not even sure that splitting up the tests is saving us time seeing as we are bottle necked by macos runners. Maybe just revert the grouping all together??

Reply:
derp. ty for reminding me about the group....

Reply:
i'm still in favor of the groups just to get some results faster in the build status, even if the overall time isn't improved due to macos runner constraint.

Reply:
^^ That makes sense!! Yeah, I guess it won't speed up successful runs, but does help "fail fast". Okay, I've been convinced to live with this overhead for at least a little bit longer, hahaha.